tests/test_ops/test_deform_conv.py

```diff
@@ -45,9 +45,19 @@ def _test_deformconv(self,
         from mmcv.ops import DeformConv2dPack
         c_in = 1
         c_out = 1
-        x = torch.tensor(input, device=device, dtype=dtype)
+        batch_size = 10
+        repeated_input = np.repeat(input, batch_size, axis=0)
+        repeated_gt_out = np.repeat(gt_out, batch_size, axis=0)
+        repeated_gt_x_grad = np.repeat(gt_x_grad, batch_size, axis=0)
+        x = torch.tensor(repeated_input, device=device, dtype=dtype)
         x.requires_grad = True
-        model = DeformConv2dPack(c_in, c_out, 2, stride=1, padding=0)
+        model = DeformConv2dPack(
+            in_channels=c_in,
+            out_channels=c_out,
+            kernel_size=2,
+            stride=1,
+            padding=0,
+            im2col_step=batch_size // 5)
```

Comment:
Shall we add two test cases: one tests `batch_size % im2col_step != 0` and the other tests  `batch_size < im2col_step`

Reply:
yes, it's necessary. I have added.